traffic_ops/client/region.go

```diff
@@ -159,3 +160,28 @@ func (to *Session) GetRegionByNamePath(name string) ([]tc.RegionName, ReqInf, er
 	}
 	return resp.Response, reqInf, nil
 }
+
+// DeleteRegion lets you DELETE a Region
+func (to *Session) DeleteRegion(id *int, name *string) (tc.Alerts, ReqInf, error) {
+	v := url.Values{}
+	if id != nil {
+		v.Add("id", strconv.Itoa(*id))
+	}
+	if name != nil {
+		v.Add("name", *name)
+	}
+	URI := apiBase + "/regions"
+	if qStr := v.Encode(); len(qStr) > 0 {
+		URI = fmt.Sprintf("%s?%s", URI, qStr)
+	}
+
+	resp, remoteAddr, err := to.request(http.MethodDelete, URI, nil)
+	reqInf := ReqInf{CacheHitStatus: CacheHitStatusMiss, RemoteAddr: remoteAddr}
+	if err != nil {
+		return tc.Alerts{}, reqInf, err
+	}
+	defer resp.Body.Close()
+	var alerts tc.Alerts
+	err = json.NewDecoder(resp.Body).Decode(&alerts)
+	return alerts, reqInf, nil
```

Comment:
You seem to be ignoring the parse error on the line above; should `return alerts, reqInf, err` instead of `return alerts, reqInf, nil`

Reply:
Returning the error now in 820659e14.